src/pages/FlowRun/Details-Tile.vue

```diff
@@ -37,8 +38,30 @@ export default {
       return FINISHED_STATES.includes(this.flowRun.state)
     },
     hasParameters() {
-      if (!this.flowRun?.parameters) return false
-      return Object.keys(this.flowRun.parameters).length > 0
+      if (!this.flowRun?.parameters && !this.flowRun?.flow?.parameters)
+        return false
+      return (
+        Object.keys(this.flowRun.parameters).length > 0 ||
+        this.flowRun.flow.parameters.length > 0
+      )
+    },
+    flowRunParams() {
+      const flowParams = this.flowRun?.flow?.parameters.reduce(
+        (accum, currentParam) => {
+          accum[currentParam.name] = currentParam.default
+          return accum
+        },
+        {}
+      )
+      const flowRunParams = this.flowRun?.parameters
+      const diff = difference(
+        Object.keys(flowParams),
+        Object.keys(flowRunParams)
+      )
+      diff.forEach(
+        differentKey => (flowRunParams[differentKey] = flowParams[differentKey])
+      )
+      return flowRunParams
```

Comment:
Couldn't we use the spread operator here? 
e.g.:
```suggestion
      return  { ...flowParams, ...this.flowRun?.parameters }
```

Reply:
Oh yes - nice idea!